Source/WebKit/UIProcess/API/Cocoa/WKNavigationDelegatePrivate.h

```diff
@@ -110,6 +110,7 @@ static const WKNavigationResponsePolicy _WKNavigationResponsePolicyBecomeDownloa
 - (void)_webView:(WKWebView *)webView didStartLoadForQuickLookDocumentInMainFrameWithFileName:(NSString *)fileName uti:(NSString *)uti;
 - (void)_webView:(WKWebView *)webView didFinishLoadForQuickLookDocumentInMainFrame:(NSData *)documentData;
 - (void)_webViewDidRequestPasswordForQuickLookDocument:(WKWebView *)webView WK_API_AVAILABLE(ios(11.0));
+- (void)_webViewDidStopRequestingPasswordForQuickLookDocument:(WKWebView *)webView WK_API_AVAILABLE(ios(WK_IOS_TBA));
```

Comment:
Slightly curious why this is not on `WKUIDelegatePrivate`, but looks like this aligns with "did request" above.

Reply:
I had the same question, but didn't want to rock the boat!